geniza/common/utils.py

```diff
@@ -32,3 +32,24 @@ def absolutize_url(local_url, request=None):
         root = root.rstrip("/")
 
     return root + local_url
+
+
+def custom_tag_string(tag_string: str) -> list:
```

Comment:
please link to the stack overflow answer if that's what this is based on

Reply:
also, no type annotations in this codebase (want to use them consistently where we do use them)

Reply:
bf36e95f57c9d1989786354e407e57572a9a80e5 ✅